mksnapshot.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/bin/sh -x
```

Comment:
I am making that -x temporarily until the CI job log can be reviewed on some snapshots made with git. When that is working the above chunk gets reverted to -e for normal use.